src/main/java/bio/terra/service/snapshotbuilder/query/FieldVariable.java

```diff
@@ -10,14 +10,22 @@ public class FieldVariable implements SQLExpression {
   private final TableVariable tableVariable;
   private final String alias;
 
+  private final boolean isDistinct;
+
   public FieldVariable(FieldPointer fieldPointer, TableVariable tableVariable) {
-    this(fieldPointer, tableVariable, null);
+    this(fieldPointer, tableVariable, null, false);
   }
 
   public FieldVariable(FieldPointer fieldPointer, TableVariable tableVariable, String alias) {
+    this(fieldPointer, tableVariable, alias, false);
+  }
+
+  public FieldVariable(
+      FieldPointer fieldPointer, TableVariable tableVariable, String alias, Boolean isDistinct) {
```

Comment:
I think this can't be `null` so it should be `boolean`
```suggestion
      FieldPointer fieldPointer, TableVariable tableVariable, String alias, boolean isDistinct) {
```